tests/perf/test_detection.py

```diff
@@ -40,6 +40,9 @@ class TestPerfObjectDetection(PerfTestBase):
         Benchmark.Model(task="detection", name="rtdetr_18", category="other"),
         Benchmark.Model(task="detection", name="rtdetr_50", category="other"),
         Benchmark.Model(task="detection", name="rtdetr_101", category="other"),
+        Benchmark.Model(task="detection", name="yolov9_s", category="other"),
+        Benchmark.Model(task="detection", name="yolov9_m", category="other"),
```

Comment:
off topic: I wonder, in what moment we will have too big list of the models to feasibly test them? :)

Reply:
I'm not sure and I think we need to replace old models with new advanced models if possible.